src/xmipp/libraries/reconstruction/resolution_pdb_bfactor.h

```diff
@@ -0,0 +1,107 @@
+/***************************************************************************
+ *
+ * Authors:    Jose Luis Vilas, 		jlvilas@cnb.csic.es
+ *
+ * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.csic.es'
+ ***************************************************************************/
+
+#ifndef _PROG_RESBFACTOR
+#define _PROG_RESBFACTOR
+
+#include <iostream>
+#include <core/xmipp_program.h>
+#include <core/xmipp_image.h>
+#include <core/metadata.h>
+#include <core/xmipp_fft.h>
+#include <core/xmipp_fftw.h>
```

Comment:
Here you need only <vector> and FileName. Remove those that are not used. Move the rest to source file.

Reply:
done